spec/models/item_spec.rb

```diff
@@ -23,12 +23,8 @@
     expect(item).to be_invalid
   end
 
-  it 'is invalid without an image' do
-    item.image = nil
-    expect(item).to be_invalid
-  end
 
-  it 'returns an empty item array' do
+  xit 'returns an empty item array' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.